tests/benchmarks/dhrystone/src/dhry_2.c

```diff
@@ -0,0 +1,191 @@
+/*
+ ****************************************************************************
+ *
+ *                   "DHRYSTONE" Benchmark Program
+ *                   -----------------------------
+ *
+ *  Version:    C, Version 2.1
+ *
+ *  File:       dhry_2.c (part 3 of 3)
+ *
+ *  Date:       May 25, 1988
+ *
+ *  Author:     Reinhold P. Weicker
+ *
+ ****************************************************************************
+ */
```

Comment:
I wonder what is the original license of this piece of code.

Reply:
Hi Jukkar,
I'm not sure about the this.
dhrystone is an ancient program and the license is undeclared on its website.

Reply:
I am not a lawyer, but if there is no known license for this code, then it might be a blocker for merging this code into zephyr.